### PR TITLE
Implements Batch Query in KairosDB wrapper

### PR DESCRIPTION
**Pull request by elgris**
This method allows sending a single request to Kairos that fetches data for multiple metric names.

### CODE REVIEW

File: tests/test_kairosdb.py [hunk 1]
```diff
@@ -145,6 +267,35 @@ def test_kairosdb_query(monkeypatch, fx_query):
     post.assert_called_with(get_final_url(), json=q)
 
 
+def test_kairosdb_query_batch(monkeypatch, fx_query_batch):
+    kwargs, res = fx_query_batch
+
+    print res
```
Comment (mohabusama) [inline tests/test_kairosdb.py]: forgotten ``print``

File: tests/test_kairosdb.py [hunk 2]
```diff
@@ -77,18 +182,39 @@ def requests_mock(resp, failure=None):
 
 
 def get_final_url():
+
     return URL + '/' + DATAPOINTS_ENDPOINT
 
 
 def get_query(kwargs):
+    kwargs_batch = dict(kwargs)
+    del kwargs_batch['name']
```
Comment (mohabusama) [inline tests/test_kairosdb.py]: Why not constructing fresh dict instead of all the ``del``s

File: tests/test_kairosdb.py [hunk 3]
```diff
@@ -116,10 +222,37 @@ def get_query(kwargs):
     return q
 
 
+def get_query_batch(kwargs):
+    start = kwargs.get('start', 5)
+    time_unit = kwargs.get('time_unit', 'minutes')
+
+    q = {'metrics': []}
```
Comment (mohabusama) [inline tests/test_kairosdb.py]: ``q = {'metrics': [kwargs['metrics']]}`` instead of two steps

**Comment by mohabusama** [comment]
👍

**Comment by vetinari** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**